evap/evaluation/templates/base.html

```diff
@@ -27,6 +27,23 @@
 </head>
 <body>
 {% block modals %}
+    <div class="modal fade" id="successMessageModal" tabindex="-1" role="dialog" aria-labelledby="successMessageModalLabel" aria-hidden="true">
+        <div class="modal-dialog" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title" id="successMessageModalLabel">{% trans "Message sent" %}</h5>
+                </div>
+                <div class="modal-body">
+                    {% trans "Your message was successfully sent." %}
+                </div>
+            </div>
+        </div>
+    </div>
```

Comment:
into `contact_modal.html`?

Reply:
i had it there first but on some pages there are multiple contact modals and that would duplicate the `successMessageModal` on the page. probably not what we want.

Reply:
as discussed, let's do it anyways.